### PR TITLE
Single Selection

### PR DESCRIPTION
**Pull request by nickwmurphy**
If you only have one item selected, display it in the dropdown title token.

Before: 
screen shot 2017-05-09 at 11 16 52 pm

After:
screen shot 2017-05-09 at 11 17 04 pm

### CODE REVIEW

File: filters/tatari/src/Tatari.scss [hunk 1]
```diff
@@ -259,3 +259,18 @@ $searchH: 26px;
...
+
+.singleSelectionDropdownTitle {
+  overflow: hidden;
+  padding: 0 5px 0 2px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-size: 10px;
+  line-height: normal;
+}
+
```
Comment (ben-burlingham) [inline filters/tatari/src/Tatari.scss]: alphabetize

Comment (nickwmurphy) [inline filters/tatari/src/Tatari.scss]: Complete 👍

File: filters/tatari/README.md [hunk 2]
```diff
@@ -4,17 +4,18 @@ A dynamically populated filter component.
...
 - The CSS file was alphabetized, and unnecessary overrides were removed
 - Many micro style tweaks were made
 - The dropdown body is now responsive with a min- and max-width
 - The sandbox file has "more realistic" data now
 - The typeahead search bar in the dropdown body now has a placeholder value
 - Deleted unnecessary CSS files
 
-### 2.2.6
-- Added internationalization for placeholders
-
```
Comment (nickwmurphy) [inline filters/tatari/README.md]: Brian ran `npm publish` after pulling in the changes from #65. Therefore, the README.md should reflect that.

File: filters/tatari/README.md [hunk 3]
```diff
@@ -4,17 +4,18 @@ A dynamically populated filter component.
...
+- Added internationalization for placeholders
 - The CSS file was alphabetized, and unnecessary overrides were removed
 - Many micro style tweaks were made
 - The dropdown body is now responsive with a min- and max-width
 - The sandbox file has "more realistic" data now
 - The typeahead search bar in the dropdown body now has a placeholder value
 - Deleted unnecessary CSS files
 
-### 2.2.6
-- Added internationalization for placeholders
```
Comment (nickwmurphy) [inline filters/tatari/README.md]: Brian ran npm publish after pulling in the changes from #65. Therefore, the README.md should reflect that.

File: filters/tatari/README.md [hunk 4]
```diff
@@ -4,17 +4,18 @@ A dynamically populated filter component.
...
+- When only one item is selected, it is displayed in the dropdown title token for better visibility
+
 #### 2.2.6
+- Added internationalization for placeholders
 - The CSS file was alphabetized, and unnecessary overrides were removed
 - Many micro style tweaks were made
 - The dropdown body is now responsive with a min- and max-width
 - The sandbox file has "more realistic" data now
 - The typeahead search bar in the dropdown body now has a placeholder value
 - Deleted unnecessary CSS files
```
Comment (nickwmurphy) [inline filters/tatari/README.md]: These changes from #65. They were pulled into #71 before published. Therefore, the README.md should show that changes from both #65 and #71 are part of the same version (`2.2.6`).

**Pull request merged**